### PR TITLE
AST-431 fix: menu close issue when resize event trigger on keyboard display

### PR DESCRIPTION
**Pull request by shubhw12**
### Description
I have added a condition check before we trigger the menu toggle logic in the window.resize event callback.

### Types of changes
Bugfix (non-breaking change which fixes an issue)

### How has this been tested?
Checked if the closes after resize?
Checked if the menu closes if the active element is input field.

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Comment by premanshup** [comment]
@shubhw12 Please update changelog in this as well for v3.5.0.

File: assets/js/unminified/frontend.js [hunk 1]
```diff
@@ -427,20 +427,28 @@ var astraTriggerEvent = function astraTriggerEvent( el, typeArg ) {
 		var menu_dropdown_close 	= document.querySelector('.menu-toggle.toggled');
 		var desktop_header_content	= document.querySelector('#masthead > #ast-desktop-header .ast-desktop-header-content');
 		var elementor_editor 		= document.querySelector('.elementor-editor-active');
-		if ( menu_dropdown_close && null === elementor_editor) {
+		// Added this var to check the current active element consition. 
+		var menu_close_flag = 'INPUT' !== document.activeElement.tagName;
```
Comment (premanshup) [inline assets/js/unminified/frontend.js:431]:
```suggestion
		var menuCloseFlag = 'INPUT' !== document.activeElement.tagName;
```

**Review by premanshup — CHANGES_REQUESTED** [review]

**Review by premanshup — APPROVED** [review]

**Comment by sagarb90** [comment]
@shubhw12 Got this console error - https://share.getcloudapp.com/7KuPGB4q please check.
I have also confirmed this error is not in `next-release`  branch.

**Review by imnavanath — APPROVED** [review]

**Review by abhishek-bsf — APPROVED** [review]

**Comment by shubhw12** [comment]
> @shubhw12 Got this console error - https://share.getcloudapp.com/7KuPGB4q please check.
> I have also confirmed this error is not in `next-release` branch. May be pull `next-release` branch will resolved this issue

I have pulled the latest code here and that resolves this console error

**Pull request merged**